rq/worker.py

```diff
@@ -221,6 +223,7 @@ def __init__(
         disable_default_exception_handler: bool = False,
         prepare_for_work: bool = True,
         serializer=None,
+        workhorse_terminated_handler: Optional[Callable[[Job, int, int, resource.struct_rusage], None]] = None
```

Comment:
Any ideas for alternative arg names? Maybe `forced_termination_handler`?

`workhorse_terminated_handler` does not imply that this callable is to handle cases where the workhorse is forcibly terminated.

Reply:
By forcibly you mean something stopped it from outside (hard/stop)? Just to clarify - We are handling a case in which the workhorse was terminated unexpectedly, while running the job. The case for stopped is handled [differently here](https://github.com/rq/rq/pull/1729/files#diff-50984f276e8a6fea5f4ba4938afb6f4fbc544fd3751fbd16027dc0a326d6405eR949).
LMKWYT

Reply:
How do you like `workerhorse_crash_handler`? See latest commit changes

Reply:
`workerhorse_crash_handler` is ok, but I'd revise is a bit to `horse_crash_handler`. What do you think about `horse_killed_handler` or `process_killed_handler`?

The term kill is quite specific to a process being killed.

Reply:
For me killed is intuitively connected to a process being killed from the outside, like when stopping or force stopping

Reply:
Crash could also mean an exception raised from inside the job though. Let's go with `work_horse_killed_handler`, I think that's the most descriptive term (and already reinforced well in the docs), we already have `send_stop_job_command` so users shouldn't mistake it for job being stopped.